Open-Source/Job-Boards/Readme.md

```diff
@@ -7,4 +7,4 @@
 - [Open Source Engineer](https://www.indeed.com/q-Open-Source-Engineer-jobs.html?vjk=1475599e24cc62aa) - Indeed
 - [Open Source Engineer](https://www.aquasec.com/about-us/careers/co/engineering/81.E1B/open-source-engineer/all/) - Aquasec
 - [RedHat Careers](https://www.redhat.com/en/jobs)
-
+- [Various job opportunities provided by outreachy sponsors](https://www.outreachy.org/opportunities/)
```

Comment:
Let's go ahead and remove this for now. You can start an issue for this, and we can discuss further how we should integrate these job openings in the repo